CRM/Activity/Form/Search.php

```diff
@@ -98,6 +98,13 @@ public function preProcess() {
     }
     else {
       $this->_formValues = $this->get('formValues');
+
+      if ($this->_force) {
+        // If we force the search then merge form values with the defaults
+        // (which include URL parameters) and set submit values to form values.
+        $this->_formValues = array_merge((array) $this->_formValues, $this->_defaults);
+        $this->_submitValues = $this->_formValues;
+      }
```

Comment:
I wonder if this block could be moved to the parent class to affect all search forms?